spec/classes/puppet_agent_spec.rb

```diff
@@ -92,6 +92,32 @@ def global_facts(facts, os)
         end
       end
     end
+
+    context 'latest' do
+      context 'valid' do
```

Comment:
I think this should be `context 'invalid'` as the catalog compilation raises error, the next one should be the `valid` context